FlowCrypt/src/androidTest/java/com/flowcrypt/email/ui/MainSignInFragmentEnterpriseFlowTest.kt

```diff
@@ -72,23 +76,38 @@ class MainSignInFragmentEnterpriseFlowTest : BaseSignTest() {
       override fun dispatch(request: RecordedRequest): MockResponse {
         val gson = ApiHelper.getInstance(getTargetContext()).gson
 
-        if (request.path?.startsWith("/api") == true) {
-          if (request.path.equals("/api/")) {
-            return handleFesAvailabilityAPI(gson)
+        when {
+          request.path?.startsWith("/api") == true -> {
+            if (request.path.equals("/api/")) {
+              return handleFesAvailabilityAPI(gson)
+            }
+
+            if (request.path.equals("/api/v1/client-configuration?domain=localhost:1212")) {
+              return handleClientConfigurationAPI(gson)
+            }
```

Comment:
we don't need `if (request.path?.startsWith("/api") == true) {` and can be written as below

```kt
request.path.equals("/api/") -> {
  return handleFesAvailabilityAPI(gson)
}
request.path.equals("/api/v1/client-configuration?domain=localhost:1212") -> {
  return handleClientConfigurationAPI(gson)
}
```